tiledb/api/examples/using_tiledb_stats.rs

```diff
@@ -0,0 +1,117 @@
+extern crate tiledb;
+use tiledb::{Array, Result as TileDBResult};
+use tiledb::config::Config;
+use tiledb::vfs::VFS;
+use tiledb::Datatype;
+
+
+const ARRAY_NAME: &str = "stats_array";
+const ATTRIBUTE_NAME: &str = "a";
+
+pub fn create_array(row_tile_extent : u32, col_tile_extent : u32) -> TileDBResult<()> {
+    let tdb = tiledb::context::Context::new()?;
+    let config: Config = tiledb::config::Config::new()?;
+    let vfs: VFS = tiledb::vfs::VFS::new(&tdb, &config)?;
+
+    let is_cur_dir = vfs.is_dir(ARRAY_NAME)?;
+    if is_cur_dir {
+        vfs.remove_dir(ARRAY_NAME)?;
+    }
+
+    let domain = {
+        let rows: tiledb::array::Dimension =
+            tiledb::array::DimensionBuilder::new::<u32>(
+                &tdb,
+                "row",
+                Datatype::UInt32,
+                &[1, 12000],
+                &row_tile_extent,
+            )?
+            .build();
+        let cols: tiledb::array::Dimension =
+            tiledb::array::DimensionBuilder::new::<u32>(
+                &tdb,
+                "col",
+                Datatype::UInt32,
+                &[1, 12000],
+                &col_tile_extent,
+            )?
+            .build();
+
+        tiledb::array::DomainBuilder::new(&tdb)?
+            .add_dimension(rows)?
+            .add_dimension(cols)?
+            .build()
+    };
+
+    let attribute_a = tiledb::array::AttributeBuilder::new(
+        &tdb,
+        ATTRIBUTE_NAME,
+        tiledb::Datatype::Int32,
+    )?
+    .build();
+
+    let schema = tiledb::array::SchemaBuilder::new(
+        &tdb,
+        tiledb::array::ArrayType::Dense,
+        domain,
+    )?
+    .add_attribute(attribute_a)?
+    .build();
+
+    tiledb::Array::create(&tdb, ARRAY_NAME, schema)
```

Comment:
Does this return an error if the array already exists?

Reply:
Yup.